benefits/core/migrations/0001_initial.py

```diff
@@ -123,6 +123,12 @@ class Migration(migrations.Migration):
                 ("active", models.BooleanField(default=False)),
                 ("jws_signing_alg", models.TextField()),
                 ("eligibility_index_intro", models.TextField()),
+                ("transit_option", models.TextField()),
+                ("benefit_action", models.TextField()),
+                ("benefit_rate", models.TextField()),
+                ("affected_service", models.TextField()),
+                ("transit_type", models.TextField()),
+                ("card_action", models.TextField()),
```

Comment:
@thekaveman @angela-tran I'm open to new names for any of these variables. I find `transit_option` and `transit_type` confusing but I wasn't sure what to name it instead, and I'm not sure how they differ.

Reply:
@thekaveman Should I limit the variables added in this PR to the ones necessary for the Home/Index pages? If so, I'd only need 1 variable and leave the rest for when we actually implement the new copy.